src/youtube-tracker.coffee

```diff
@@ -2,11 +2,10 @@
 #   A Hubot script for tracking YouTube
 #
 # Configuration:
-#   LIST_OF_ENV_VARS_TO_SET
+#   GOOGLE_API_KEY is required to make use of this script.
 #
 # Commands:
-#   hubot hello - <what the respond trigger does>
-#   orly - <what the hear trigger does>
+#   <none>
```

Comment:
Not important, but be sure to include command here as it is used for help documentation.